### PR TITLE
Integrate xcm configuration and benchmarks

### PR DESCRIPTION
**Pull request by drgora**

### CODE REVIEW

File: runtime/src/xcm_config.rs [hunk 1]
```diff
@@ -510,6 +244,8 @@ impl xcm_executor::Config for XcmConfig {
     type HrmpChannelClosingHandler = ();
 }
 
+const FELLOWSHIP_ADMIN_INDEX: u32 = 1; // to be moved to some constants mod
```
Comment (DanieleDiBenedetto) [inline runtime/src/xcm_config.rs:247]: What is this ? Is this mandatory ?

**Review by DanieleDiBenedetto — APPROVED** [review]
Thanks for the amazing work !

File: runtime/src/xcm_config.rs (hunk 1 above)
Comment (drgora) [reply runtime/src/xcm_config.rs:247]: No it isn't, as we are not using any of these as of today. I also liked the idea of removing this, but then I thought to keep it consistent with the rest of the runtime, that is with the _governance_ part in this case.

File: relay-node/tests/purge_chain_works.rs [hunk 2]
```diff
@@ -36,7 +36,7 @@ const PARITY: &str = "paritydb/full";
 
 #[tokio::test]
 async fn purge_chain_rocksdb_works() {
-    run_with_timeout(Duration::from_secs(10 * 60), async move {
+    run_with_timeout(Duration::from_secs(20 * 60), async move {
```
Comment (la10736) [inline relay-node/tests/purge_chain_works.rs:39]: Maybe 5 is enough

File: relay-node/tests/purge_chain_works.rs [hunk 3]
```diff
@@ -80,7 +80,7 @@ async fn purge_chain_rocksdb_works() {
 
 #[tokio::test]
 async fn purge_chain_paritydb_works() {
-    run_with_timeout(Duration::from_secs(10 * 60), async move {
+    run_with_timeout(Duration::from_secs(20 * 60), async move {
```
Comment (la10736) [inline relay-node/tests/purge_chain_works.rs:83]: See above comment

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -369,19 +370,21 @@ impl pallet_timestamp::Config for Runtime {
...
-pub const EXISTENTIAL_DEPOSIT: u128 = MILLICENTS;
+parameter_types! {
+    /// Existential deposit.
+    pub const ExistentialDeposit: Balance = EXISTENTIAL_DEPOSIT;
+}
 
 impl pallet_balances::Config for Runtime {
     type MaxLocks = ConstU32<50>;
-    type MaxReserves = ();
+    type MaxReserves = ConstU32<50>;
```
Comment (la10736) [inline runtime/src/lib.rs:380]: Just curiosity ... why?

**Review by la10736 — APPROVED** [review]
LGTM.... Just reduce the test timeout to 5 minutes

File: relay-node/tests/purge_chain_works.rs (hunk 2 above)
Comment (drgora) [reply relay-node/tests/purge_chain_works.rs]: Right, let's make it `5`

File: relay-node/tests/purge_chain_works.rs (hunk 3 above)
Comment (drgora) [reply relay-node/tests/purge_chain_works.rs]: Done

File: runtime/src/lib.rs (hunk 4 above)
Comment (drgora) [reply runtime/src/lib.rs:380]: I was using this to test some `xcm` scenarios. Anyways, I will add some e2e tests to cover the basic xcm interactions that we want to support, and then remove this if unneeded.

**Pull request merged**